scripts/travis/before_script.sh

```diff
@@ -1,6 +1,6 @@
 set -ex
 
-sh /etc/init.d/xvfb start
+# sh /etc/init.d/xvfb start
```

Comment:
Do you mean to be commenting this out?

Reply:
Yes. __services: - xvfb__ makes in unnecessary. See:
https://docs.travis-ci.com/user/gui-and-headless-browsers/#using-xvfb-to-run-tests-that-require-a-gui

Reply:
So why not just delete the line?